src/ledger/backend/ethereumOnChainQuerier.ts

```diff
@@ -1,9 +1,10 @@
-import { providers, Signer } from "ethers";
+import { providers, Signer, BigNumber } from "ethers";
 
 import { IERC721__factory } from ".";
-import { Tokens } from "../assets";
+import { mapNFTs, Tokens } from "../assets";
```

Comment:
```suggestion
import { Tokens } from "../assets";
```
the import is not used.